tools/packaging/kata-deploy/local-build/Makefile

```diff
@@ -163,6 +163,17 @@ rootfs-initrd-tarball: agent-tarball
 
 runk-tarball: copy-scripts-for-the-tools-build
 	${MAKE} $@-build
+rootfs-nvidia-gpu-image-tarball: #agent-tarball busybox-tarball
```

Comment:
Why adding the dependencies commented out?
Ditto for all the other targets added.

Reply:
Local builds :) Forgot to add them in